Makefile

```diff
@@ -9,14 +9,18 @@ VERSION="0.5.0"
 CFLAGS ?= -Wall -Wextra -Wno-unused-parameter -Wno-parentheses -Wno-format-overflow
 override CFLAGS += -DVERSION=\"$(VERSION)\"
 
-VPATH=$(WAYLAND_PROTOCOLS_DIR)/stable/xdg-shell:$(WAYLAND_PROTOCOLS_DIR)/unstable/xdg-decoration
 LIBS=-lrt -lm -lutil -lwayland-client -lwayland-cursor -lxkbcommon -Ltsm -lhtsm
-OBJ=xdg-shell.o xdg-decoration-unstable-v1.o gtk-primary-selection.o glyph.o main.o
-GEN=xdg-shell.c xdg-shell.h xdg-decoration-unstable-v1.c \
-	xdg-decoration-unstable-v1.h gtk-primary-selection.c gtk-primary-selection.h
+OBJ=xdg-shell.o \
+		xdg-decoration-unstable-v1.o \
+		primary-selection-unstable-v1.o \
+		glyph.o \
+		main.o
+GEN=xdg-shell.c xdg-shell.h \
+		xdg-decoration-unstable-v1.c xdg-decoration-unstable-v1.h \
+		primary-selection-unstable-v1.h primary-selection-unstable-v1.c 
```

Comment:
Nitpicks but: It should be lined up at tabstop 8, not 2. And there's a trailing space.